caplena/endpoints/projects_endpoint.py

```diff
@@ -677,6 +708,32 @@ def parse_obj(cls, obj: Dict[str, Any]) -> "RowsAppend":
         return super().parse_obj(obj)
 
 
+class RowsAppendStatus(BaseObject[ProjectsController]):
+    """Status of requested task and it's subtasks"""
+
+    class MeerkatSubTaskStatus(BaseObject[ProjectsController]):
+        """Subtask and it's sub-subtasks status"""
+
+        __fields__ = {"id", "status", "subtasks"}
+
+        id: uuid.UUID
+        """ID of the subtask"""
+
+        status: Optional[Literal["in_progress", "succeeded", "failed", "debounced", "timed_out"]]
+        """Status of the subtask"""
+
+        subtasks: Optional[List[Self]]
+        """subtasks of the task"""
+
+    __fields__ = {"status", "tasks"}
+
+    status: Literal["in_progress", "succeeded", "failed", "debounced", "timed_out"]
```

Comment:
@pazembrz  as discussed in the meeting, we shouldn't expose "debounced". also what is the difference between "timed_out" and "failed"?

Reply:
`denounced` is now removed as labelhippo won't return this status anymore (it will return "succeeded" instead)

`failed` status is when task itself will fail (raise an exception for example)
`timed_out` is when task in meerkat will be registered with timeout, and this timeout will pass without changing task status to `succeeded` or `failed`